src/transformers/example_rumble_kong_league/main.py

```diff
@@ -52,15 +55,16 @@ def entrypoint(self, txn) -> None:
             if event["decoded"] is None:
                 logging.warning(f"No name for event: {event}")
                 continue
-
-            if event["decoded"]["name"] == "Transfer":
+            
```

Comment:
empty space. Not sure how black allowed this. Did you install the pre commit as per the readme of the repo?